src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,104 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void unit() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User alice = new User();
+        alice.setPassword("qwerty");
+        alice.setAge(25);
+        alice.setLogin("alice123");
+        User bob = new User();
+        bob.setPassword("123456");
+        bob.setAge(18);
+        bob.setLogin("hamster18");
+        User actualAlice = registrationService.register(alice);
+        assertEquals(alice, actualAlice);
+        User actualBob = registrationService.register(bob);
+        assertEquals(bob, actualBob);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RuntimeException.class, ()
+                -> registrationService.register(null));
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        User nullLoginUser = new User();
```

Comment:
```suggestion
        User user = new User();
```
It is better to use just that name, as the name of the method makes everything clear. Fix it everywhere